src/internal/opts.rs

```diff
@@ -72,4 +72,7 @@ pub struct FdOptions {
 
     /// Constraints on last modification time of files
     pub time_constraints: Vec<TimeFilter>,
+
+    /// Whether to print errors on stdout
+    pub verbose: bool,
```

Comment:
I suggest that we rename all variables from `verbose` to `show_filesystem_errors`.